websites/B/presence.ts

```diff
@@ -0,0 +1,257 @@
+      } else if (path.pathname.includes ("demographics")) {
+        presenceData.details = "Viewing on page creator";
+        presenceData.state = "Insights: demographics";
+      } 
+    } else {
+      presenceData.details = "Viewing on Insights";
+      presenceData.state = "Insights: all";
+      if (path.pathname.includes ("followers")) {
+        presenceData.details = "Viewing on Insights";
+        presenceData.state = "Insights: followers";
+      } else if (path.pathname.includes ("demographics")) {
+        presenceData.details = "Viewing on Insights";
+        presenceData.state = "Insights: demographics";
+      } 
+    }
+  } else if (path.pathname.includes ("user")) {
+    const user = document.querySelector("head > title").textContent ?? "Unknow username";
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = "Viewing on profile";
+    presenceData.state = user;
+  } else if (path.pathname.includes ("terms")) {
+    presenceData.startTimestamp = browsingStamp;
+    if (path.pathname.includes ("ads")) {
+      presenceData.details = "Terms of ads and boosts";
+      presenceData.smallImageKey = "reading";
+    } else {
+      presenceData.details = "Privacy Policy";
+      presenceData.smallImageKey = "reading";
+    }
+  } else if (path.pathname.includes ("faqs")) {  
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = "FAQS";
+    presenceData.smallImageKey = "reading";
+  } else if (path.pathname.includes ("brand")) {
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = "Guild to use brand";
+    presenceData.smallImageKey = "reading";
+  } else if (path.pathname.includes ("search")) {
+    const searchres = document.querySelector("#__next > div > div.container.flex-grow-1.d-flex.flex-column > div > div.jsx-3959159148.col-12.col-md-8.col-lg-6.col-xl-5.px-0 > div > div:nth-child(2) > div > form > div.jsx-1849148460.d-flex.align-items-center > label > input").getAttribute("value") ?? "Unknow search";
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = "Searching for:";
+    presenceData.state = searchres;
+    presenceData.smallImageKey = "search";
+  } else if (path.pathname.includes ("bookmark")) {
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = "Viewing on bookmark";
+  } else if (path.pathname.includes ("popular")) {
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = "Viewing on popular page";
+    presenceData.state = "Trending";
+  }
+  else {
+    const titlen = document.querySelector("head > title").textContent ?? "Unknow page";
+    presenceData.details = "Viewing on page";
+    presenceData.state = titlen;
+    presenceData.startTimestamp = browsingStamp;
+  }
+}
+
+if (presenceData.details == null) {
+  presence.setTrayTitle();
+  presence.setActivity();
+} else {
+  presence.setActivity(presenceData);
+  //console.log(presenceData);
+}
+});
+
+
```

Comment:
Pretty sure this file shouldnt be here xD

Reply:
websites/B/presence.ts

Reply:
OOF Sorry